### DISCUSSION TITLE
Adding relation fields to an existing collection

### DISCUSSION THREAD

**Discussion by DoormatIka** [pocketbase/pocketbase#2156] [Q&A] [answered]
I'm using Javascript's SDK.
The documentation doesn't have anything relating to adding relation fields to an existing collection so I'm stuck on how to do this..
```js
const subj = await pb
    .collection("subjects")
    .getOne(id); // selecting an id to insert my relation fields on
```

**Comment by ganigeorgiev** [comment]
Your example returns a record with the specified id, doesn't return a collection.

I'm not sure that I understand what do you mean. Could you elaborate what you are trying to do?

**Reply by DoormatIka** [reply]
I'm sorry for the unclear question..
I'm trying to insert a Record into a relation field of another Record that I selected using its id.
![image](https://user-images.githubusercontent.com/68234036/227756344-fa5a8380-5192-45c6-a862-e552eba800fc.png)

How do I do that with Pocketbase?

**Comment by DoormatIka** [comment, marked as answer]
I'll answer my own question here.
To add relation fields, create a new record:
```js
const topic = await locals.pb
        .collection("topics")
        .create(req);
```
Then pass it in an existing record:
```js
const subj = await locals.pb
   .collection("subjects")
   .update(subject_id, { "field": [`${topic.id}`] })
```
